pkg/systemlogmonitor/README.md

```diff
@@ -96,20 +96,20 @@ default.
 Temporary problems will be reported as counter metrics, such as below example:
 
 ```
-# HELP problem_counter Number of times a specific type of problem have occurred.
-# TYPE problem_counter counter
-problem_counter{reason="TaskHung"} 2
+# HELP problem_count_total Number of times a specific type of problem have occurred.
+# TYPE problem_count_total counter
+problem_count_total{reason="TaskHung"} 2
 ```
 
 Permanent problems will be reported as both gauge metrics and counter metrics, such as below
 example:
 
 ```
-# HELP problem_counter Number of times a specific type of problem have occurred.
-# TYPE problem_counter counter
-problem_counter{reason="DockerHung"} 1
-# HELP problem_gauge Whether a specific type of problem is affecting the node or not.
-# TYPE problem_gauge gauge
-problem_gauge{condition="KernelDeadlock",reason="DockerHung"} 1
+# HELP problem_count_total Number of times a specific type of problem have occurred.
+# TYPE problem_count_total counter
+problem_count_total{reason="DockerHung"} 1
+# HELP problem_state Whether a specific type of problem is affecting the node or not.
+# TYPE problem_state gauge
+problem_state{condition="KernelDeadlock",reason="DockerHung"} 1
```

Comment:
Why there is a label `condition`? Should it be `type`?

Reply:
Good catch! Yes this is a typo. Fixed :)